doc/how_to_guides/isaac_panda/.docker/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM osrf/ros:humble-desktop-jammy
+
+SHELL ["/bin/bash", "-c", "-o", "pipefail"]
+
+RUN echo "deb [trusted=yes] https://raw.githubusercontent.com/moveit/moveit2_packages/jammy-humble/ ./" \
+    | sudo tee /etc/apt/sources.list.d/moveit_moveit2_packages.list
+RUN echo "yaml https://raw.githubusercontent.com/moveit/moveit2_packages/jammy-humble/local.yaml humble" \
+    | sudo tee /etc/ros/rosdep/sources.list.d/1-moveit_moveit2_packages.list
+
+# Bring the container up to date to get the latest ROS2 humble sync on 01/27/23
+# hadolint ignore=DL3008, DL3013
+RUN apt-get update && apt-get upgrade -y && rosdep update
+
+# Install packages required to run the demo
+RUN apt-get install -y --no-install-recommends \
+    ros-humble-moveit
+    # ros-humble-moveit-resources
+
+# Create Colcon workspace and clone the needed source code to run the demo
+RUN mkdir -p /root/isaac_moveit_tutorial_ws/src
+WORKDIR /root/isaac_moveit_tutorial_ws/src
+RUN git clone https://github.com/PickNikRobotics/topic_based_ros2_control.git
+RUN git clone https://github.com/JafarAbdi/moveit_resources.git -b feature/update-controller-name
+# Docker does not allow copying in parent directories from the host so grab another copy of the tutorials
+RUN git clone https://github.com/MarqRazz/moveit2_tutorials.git -b pr-isaac_tutorial --depth 1
```

Comment:
Do you want to update this like we discussed to point at humble or use binaries?